WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -123,6 +134,63 @@ function WeakAuras.regionPrototype.AddProperties(properties, defaultsForRegion)
   end
 end
 
+local function SetFrameGlow(self, options)
```

Comment:
You’re duplicating code. The better option is to expose the original code somehow and reuse it if you can. That way if we ever need to fix a bug in this we only have to do it once.